### PR TITLE
Wave dashing nerfed

### PR DESCRIPTION
**Pull request by optimumtact**
Thank you PkPenguin

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
fuck you

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -380,6 +380,9 @@ Sorry Giacom. Please don't be mad :(
 	set category = "IC"
 
 	resting = !resting
+	if(resting)//prevents wavedashing
+		src << "You start to get to your feet"
```
Comment (Iamgoofball) [inline code/modules/mob/living/living.dm]: .

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -380,6 +380,9 @@ Sorry Giacom. Please don't be mad :(
...
+		sleep(3)
```
Comment (Iamgoofball) [inline code/modules/mob/living/living.dm]: this is 3 deciseconds

**Comment by PKPenguin321** [comment]
NO

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -380,6 +380,9 @@ Sorry Giacom. Please don't be mad :(
 	set category = "IC"
 
 	resting = !resting
+	if(resting)//prevents wavedashing
```
Comment (Iamgoofball) [inline code/modules/mob/living/living.dm]: put this before the resting = !resting shit

**Comment by KorPhaeron** [comment]
This would kill the pro ss13 scene

**Pull request closed without merging**